CONTRIBUTING.md

```diff
@@ -60,25 +62,25 @@ This section describes how to build and run the Botkube from the source code.
      docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
      docker push <your_account>/botkube:v9.99.9-dev
      ```
-     Where `<your_account>` is Docker hub account to which you can push the image.
+     Where `<your_account>` is Docker hub or any other registry provider account to which you can push the image.
+
+2. Ensure you have a Slack installation already setup:
 
-2. Deploy the newly created image in your cluster:
+   Please refer to the [Slack installation instructions](https://botkube.io/docs/installation/socketslack/) to ensure you have a valid integration.
 
+3. Deploy the newly created image in your cluster:
+   
+   Use [the instructions here](https://botkube.io/docs/installation/socketslack/#install-botkube-backend-in-kubernetes-cluster) to Helm install Botkube while updating the following flags:  
```

Comment:
(Cannot add it as a single suggestion, but if you agree with the suggestion above, please apply this and it will automatically merge these sections)
```suggestion

```